bahmnicore-api/src/main/java/org/bahmni/module/bahmnicore/contract/form/data/Provider.java

```diff
@@ -0,0 +1,40 @@
+package org.bahmni.module.bahmnicore.contract.form.data;
```

Comment:
why "org.bahmni....form.data" package? just creating under "org.bahmni...contract.form" should suffice

Reply:
actually if the "Provider" is common model, then it should be under "model" package. and since we do not create provider as such and this is just  value object, this should not be under any "contract" package at all.